### DISCUSSION TITLE
Stable update channel

### DISCUSSION THREAD

**Discussion by shuaixr** [Mygod/VPNHotspot#379] [Ideas]
I want to receive only stable update alerts

**Comment by Mygod** [comment]
Congratulations! This is now the stable update channel! Go subscribe nau!!!

**Comment by Mygod** [comment]
https://github.com/Mygod/VPNHotspot/releases/tag/v2.14.2

**Comment by Mygod** [comment]
https://github.com/Mygod/VPNHotspot/releases/tag/v2.14.4

**Comment by Mygod** [comment]
https://github.com/Mygod/VPNHotspot/releases/tag/v2.15.1

**Comment by Mygod** [comment]
https://github.com/Mygod/VPNHotspot/releases/tag/v2.15.2

**Comment by Mygod** [comment]
https://github.com/Mygod/VPNHotspot/releases/tag/v2.15.3

**Comment by Mygod** [comment]
https://github.com/Mygod/VPNHotspot/releases/tag/v2.15.5

**Comment by Mygod** [comment]
# As previously announced, this release does not support Android 5-8.1. Use v2.15 instead.

https://github.com/Mygod/VPNHotspot/releases/tag/v2.16.5

**Comment by Mygod** [comment]
https://github.com/Mygod/VPNHotspot/releases/tag/v2.16.6

**Comment by Mygod** [comment]
https://github.com/Mygod/VPNHotspot/releases/tag/v2.16.8

**Comment by Mygod** [comment]
https://github.com/Mygod/VPNHotspot/releases/tag/v2.16.9

**Comment by Mygod** [comment]
https://github.com/Mygod/VPNHotspot/releases/tag/v2.16.10

**Comment by Mygod** [comment]
https://github.com/Mygod/VPNHotspot/releases/tag/v2.16.11

**Comment by Mygod** [comment]
https://github.com/Mygod/VPNHotspot/releases/tag/v2.17.7

**Comment by Mygod** [comment]
https://github.com/Mygod/VPNHotspot/releases/tag/v2.17.10

Oops

**Comment by Mygod** [comment]
https://github.com/Mygod/VPNHotspot/releases/tag/v2.18.0

**Comment by Mygod** [comment]
https://github.com/Mygod/VPNHotspot/releases/tag/v2.18.2

**Comment by Mygod** [comment]
You can now use [Obtainium](https://apps.obtainium.imranr.dev/redirect.html?r=obtainium%3A%2F%2Fapp%2F%257B%2522id%2522%253A%2522be.mygod.vpnhotspot%2522%252C%2522url%2522%253A%2522https%253A%252F%252Fgithub.com%252FMygod%252FVPNHotspot%2522%252C%2522author%2522%253A%2522%2522%252C%2522name%2522%253A%2522%2522%252C%2522additionalSettings%2522%253A%2522%257B%255C%2522includeZips%255C%2522%253Atrue%252C%255C%2522apkFilterRegEx%255C%2522%253A%255C%2522zip%255C%2522%257D%2522%257D) instead to get automatic updates for stable channel.
